packages/mobile/src/backup/BackupQuiz.tsx

```diff
@@ -0,0 +1,294 @@
+          <View style={styles.chosenWordsContainer}>
+            {_.times(mnemonicLength, (i) => (
+              <Text
+                key={`selected-word-${i}`}
+                style={[styles.chosenWord, userChosenWords[i] && styles.chosenWordFilled]}
+              >
+                {(userChosenWords[i] && userChosenWords[i].word) || i + 1}
+              </Text>
+            ))}
+          </View>
+          <Text style={styles.bodyText}>{t('backupQuizInfo')}</Text>
+          {!isQuizComplete && (
+            <Text style={styles.bodyTextBold}>
+              {t('backupQuizWordCount', { index: currentWordIndex, total: mnemonicLength })}
+            </Text>
+          )}
+          <View style={styles.mnemonicButtonsContainer}>
+            {mnemonicWordsToDisplay.map((word, index) => (
+              <SmallButton
+                key={'mnemonic-button-' + word}
+                solid={false}
+                text={word}
+                style={styles.mnemonicWordButton}
+                onPress={this.onPressMnemonicWord(word, index)}
+              />
+            ))}
+          </View>
+          <View style={styles.backButtonsContainer}>
+            {currentWordIndex > 1 && (
+              <SmallButton
+                onPress={this.onPressBackspace}
+                solid={false}
+                text={this.props.t('global:goBack')}
+                style={styles.backButton}
+                textStyle={fontStyles.link}
+              >
+                <Backspace color={colors.celoGreen} />
+              </SmallButton>
+            )}
+            {isQuizComplete && (
+              <Link onPress={this.onPressReset} style={styles.resetButton}>
+                {t('global:reset')}
+              </Link>
+            )}
+          </View>
+        </ScrollView>
+        {isQuizComplete && (
+          <Button
+            onPress={this.onPressSubmit}
+            text={t('global:submit')}
+            standard={false}
+            type={BtnTypes.PRIMARY}
+            testID={'QuizSubmit'}
+          />
+        )}
+      </SafeAreaView>
+    )
+  }
+}
+
+export default componentWithAnalytics(
+  connect<{}, DispatchProps, {}, RootState>(
+    null,
+    { setBackupCompleted, showError }
+  )(withNamespaces(Namespaces.backupKeyFlow6)(BackupQuiz))
+)
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: colors.background,
+    justifyContent: 'space-between',
+  },
+  scrollContainer: {
+    flexGrow: 1,
+    paddingHorizontal: 20,
+    paddingBottom: 20,
+  },
+  bodyText: {
+    marginTop: 20,
+    ...fontStyles.bodySecondary,
+    color: colors.lightGray,
+    textAlign: 'center',
+  },
+  bodyTextBold: {
+    ...fontStyles.body,
+    ...fontStyles.semiBold,
+    textAlign: 'center',
+    marginTop: 25,
+  },
+  chosenWordsContainer: {
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    justifyContent: 'center',
+  },
+  chosenWord: {
+    ...fontStyles.bodySmall,
```

Comment:
The text doesn't look vertically centered in the container because of #1242 
The workaround for now is to set `lineHeight: undefined` where we can.

Reply:
Will do, thanks!